projects/schematics/src/shared/utils/workspace-utils.ts

```diff
@@ -177,7 +179,11 @@ export function validateSpartacusInstallation(packageJson: any): void {
 }
 
 export function scaffoldStructure(options: SpartacusOptions): Rule {
-  return (_tree: Tree) => {
+  return (_tree: Tree, context: SchematicContext) => {
+    if (options.debug) {
+      context.logger.info(`⌛️ Scaffolding Spartacus file structure...`);
```

Comment:
maybe throughout the application, you should make use of debugLog as mentioned previously.

You used it in line 207 here, so maybe change all instances to use that of 

```
debugLog(xxx/formatFeatureStart/formatFeatureComplete, options.debug);

```

Reply:
I updated all the instances that I could.
Note that the new `debugLogRule` is just a rule that logs the given message, and we can't use it in places where we don't have a chain of rules being returned.